mapo/agents/mapo/off_mapo_policy.py

```diff
@@ -10,15 +10,24 @@
 from mapo.agents.mapo.mapo_policy import MAPOTFPolicy
 
 
+def _build_dynamics_loss(policy, batch_tensors):
```

Comment:
I guess this function would be better called `_build_dynamics_mle_loss()` ...

Reply:
Good point, we can switch between different losses this way.